app/forms/base_form.rb

```diff
@@ -102,7 +102,7 @@ def set_instance_variables_for_attributes_if_not_set_but_in_model(attrs:, model_
 
       attrs.map(&:to_s).each do |method|
         model_value = model_attributes[method]
-        instance_variable_set(:"@#{method}", model_value) if model_value && attributes[method].nil?
+        instance_variable_set(:"@#{method}", model_value) if !model_value.nil? && attributes[method].nil?
```

Comment:
I had to make this change for boolean.
When `model_value` is `false` and `attributes[method]` is `nil`